### PR TITLE
TextField component

### PR DESCRIPTION
**Pull request by MnsrIl**

### CODE REVIEW

**Comment by oetherington** [comment]
Nice work!

The only issue is that the translation.json files are generated automatically by parsing the code and finding all the string literals that are passed into the `t` function, so it won't be able to find the strings here.

The easiest solution is to just add a dummy function `const t = (s) => s` to the top of [src/pages/poland-form/utils.js](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/pull/78/files#diff-6205e3c4d9a9ad72492e59f161f36d91ad97b8b41d6572eabfbedc856d924be3), and then we can use it to wrap the titles in the config, so `{ name: "firstName", title: "First Name" }` becomes `{ name: "firstName", title: t("First Name") }`.

File: src/Components/TextField/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from "react-i18next";
+
+const TextField = ({ name, onChange, value = "", title }) => {
+    const { t } = useTranslation();
+
+    return (
+        <label className="block text-gray-700 text-sm font-bold mb-5" htmlFor={name}>
+            {title && <p>{t(title)}</p>}
```
Comment (MnsrIl) [inline src/Components/TextField/index.js:8]: Actually, we translate the TextField title by this line. Or it won't work with new changes with translating?

Comment (oetherington) [reply src/Components/TextField/index.js:8]: Yes, it's fine for us to do the translation here, but the translation library can only add translations to the json files if a string literal is parsed into `t`, not a variable, so we need to add this dummy function.

You can see we did something similar [here](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/main/src/hooks/useCountryData.js#L7).

If you'd prefer, I can just add this myself and you can take a look after?

Comment (MnsrIl) [reply src/Components/TextField/index.js:8]: Oh, I got this, thanks :)
I'll fix this, but should we then replace this:
````javascript
{title && <p>{t(title)}</p>}
````
to: 
````javascript
{title && <p>{title}</p>}
````
?

**Comment by oetherington** [comment]
Looks great! Thanks!

**Pull request merged**